lldb/source/Plugins/Language/CPlusPlus/GenericBitset.cpp

```diff
@@ -28,7 +28,7 @@ class GenericBitsetFrontEnd : public SyntheticChildrenFrontEnd {
 
   GenericBitsetFrontEnd(ValueObject &valobj, StdLib stdlib);
 
-  size_t GetIndexOfChildWithName(ConstString name) override {
+  llvm::Expected<size_t> GetIndexOfChildWithName(ConstString name) override {
     return formatters::ExtractIndexFromString(name.GetCString());
   }
```

Comment:
`lldb_private::formatters::ExtractIndexFromString` will also need to be changed to return `llvm::Expected`

Reply:
I plan on doing this in a separate patch. For now, I have added code which checks if `lldb_private::formatters::ExtractIndexFromString` returns `UINT32_MAX`.

Reply:
Added to the list of tasks to do in another PR, in this PR's description.